holoviews/operation/element.py

```diff
@@ -899,8 +899,8 @@ def _process_layer(self, element, key=None):
         if element.interface not in column_interfaces:
             element = element.clone(tuple(element.columns().values()))
 
-        xstart, xend = self.p.x_range if self.p.x_range else element.range(0)
-        ystart, yend = self.p.y_range if self.p.y_range else element.range(1)
+        xstart, xend = element.range(0)
```

Comment:
This doesn't seem correct, it is important that you can pass in the required `x_range` and `y_range`